package.json

```diff
@@ -80,9 +79,9 @@
   },
   "peerDependencies": {
     "cnvs": "1.1.x",
-    "react": "^15.0.0",
+    "react": "^15.0.0 || ^16.0.0",
```

Comment:
Not sure how I feel about having `||` here but well.

Reply:
Hey, @weblancaster, what is your concern? We're testing against _all_ versions + I wanted to keep upgrades smooth and have a fallback option. Is it the way I expressed the supported range or having both versions in first place is the concern?